.travis.yml

```diff
@@ -17,25 +17,17 @@ env:
     - CHECK_TYPE="install"
     - INSTALL_TYPE="pip"
     - INSTALL_DEPENDS="pip setuptools"
-  matrix:
+  jobs:
     - CHECK_TYPE="style"
     - CHECK_TYPE="test"
     - INSTALL_TYPE="sdist"
     - INSTALL_TYPE="wheel"
-    - INSTALL_DEPENDS="pip==18.1 setuptools==30.2.1"
-    - INSTALL_DEPENDS="pip==10.0.1 setuptools==30.4.0"
+    - INSTALL_DEPENDS="pip==19.0 setuptools==30.2.1"
```

Comment:
this should do it?
```suggestion
    - INSTALL_DEPENDS="pip==19.0 setuptools==40.8.0"
```

Reply:
Oh, I see. Yeah, the issue here was verifying that `pip` should be able to install the necessary setuptools to perform the build. If `imagecodecs` is making this impossible, then we can just remove this check.

Alternately, we can change this to `BUILD_DEPENDS`, make sure the wheel builds (dependencies only need to be built on install, not build), and then install it with latest pip/setuptools.